examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -0,0 +1,85 @@
+"""
+==================
+Image Antialiasing
+==================
+
+Images are represented by discrete pixels, either on the screen or in an
+image file.  When data that makes up the image has a different resolution
+than its representation on the screen we will see aliasing effects.
+
+By default Matplotlib has a mild anti-aliasing filter turned on for images
```

Comment:
Maybe:
> The default image interpolation in Matplotlib is 'antialiased'. This uses a hanning interpolation for reduced aliasing in most situations. Only when there is upsampling by a factor of 2 or >=3 a simple nearest neighbor interpolation is used.

Reply:
Changed to 
```
The default image interpolation in Matplotlib is 'antialiased'.  This uses a
hanning interpolation for reduced aliasing in most situations. Only when there
is upsampling by a factor of 1, 2 or >=3 is 'nearest' neighbor interpolation
used.
```

Note that factor of 1 sampling is also special cased (not strictly upsampling, but...)